### PR TITLE
Marvell AC5x DB - Add single CPU Slim mode of operation

### PR DESCRIPTION
**Pull request by eladnachman**
This Pull request and series of commits introduce a new Marvell switchdev mode of operation - the Slim mode of operation, which allows operating the switchdev on boards with only a single CPU.

Currently, this mode is supported for the Marvell AC5x SOC which combines switching Packet Processor and CPU on a single die, for the Marvell DB (Development Board).

### CODE REVIEW

File: packages/platforms/marvell/arm64/ac5x-db/onlp/builds/arm64_marvell_ac5x_db/arm64_marvell_ac5x_db.mk [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+###############################################################################
+#
+# Inclusive Makefile for the arm64_marvell_ac5x_db module.
+#
+# Autogenerated 2022-02-08 15:37:45.681643
```
Comment (paulmenzel) [inline packages/platforms/marvell/arm64/ac5x-db/onlp/builds/arm64_marvell_ac5x_db/arm64_marvell_ac5x_db.mk:6]: What command autogenerated these?

**Review by paulmenzel — CHANGES_REQUESTED** [review]
1.  GitHub shows, that some files do not use UNIX line endings. It’d be great to have that fixed.
2.  The last commit should be squashed into the commit with the typo.

File: packages/platforms/marvell/arm64/ac5x-db/onlp/builds/arm64_marvell_ac5x_db/arm64_marvell_ac5x_db.mk (hunk 1 above)
Comment (eladnachman) [reply packages/platforms/marvell/arm64/ac5x-db/onlp/builds/arm64_marvell_ac5x_db/arm64_marvell_ac5x_db.mk]: Removed from Commit

**Comment by eladnachman** [comment]
> 1. GitHub shows, that some files do not use UNIX line endings. It’d be great to have that fixed.
> 2. The last commit should be squashed into the commit with the typo.

Done

**Review by taraschornyiplv — DISMISSED** [review]

**Review by taraschornyiplv — APPROVED** [review]

**Comment by robimarko** [comment]
@taraschornyiplv Since you tagged me, can you explain how does this work?
I see that CPSS kmods are added for using DMA and interrupts from userspace.

**Review by robimarko — APPROVED** [review]

**Review by taraschornyiplv — APPROVED** [review]

**Pull request merged**